Assets/Scripts/2D/GuiManagerScript.cs

```diff
@@ -247,7 +251,7 @@ public void DeregisterLoadWorldPostProgressOp(PostProgressOperation op)
         _loadWorldPostProgressOp -= op;
     }
 
-    public void ResetAllDialogs()
+    private void ResetAllDialogs()
```

Comment:
Why were these set to private?

Reply:
Those function were not intended to be used outside of the class. Beside that, if they are public they become visible to the Unity3D editor, which is not really good when the class way too many public functions. Unity3D is not designed as IDE and it sucks at discoverability. It becomes a chore trying to find specific functions in unity if there are too many of them exposed.